### PR TITLE
Marathon SD: Add port def/mapping labels

### PR DESCRIPTION
**Pull request by mattbostock**
These labels were added in:
https://github.com/prometheus/prometheus/pull/2506

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -716,6 +716,8 @@ The following meta labels are available on targets during [relabeling](#relabel_
 * `__meta_marathon_image`: the name of the Docker image used (if available)
 * `__meta_marathon_task`: the ID of the Mesos task
 * `__meta_marathon_app_label_<labelname>`: any Marathon labels attached to the app
+* `__meta_marathon_port_definition_label_<labelname>`: any labels attached to the port definitions
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: "The labels of the port defintions" would be in line with the rest of the doc

**Comment by mattbostock** [comment]
Is `master` the correct branch for this change as the labels being documented here are already in released versions of Prometheus?

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (mattbostock) [inline content/docs/operating/configuration.md]: Would 'the port definition labels' read better?

**Comment by brian-brazil** [comment]
Yes, master is correct

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: I'm not too bothered either way

**Comment by mattbostock** [comment]
Amended.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**